src/test/regress/pg_regress_multi.pl

```diff
@@ -105,6 +107,7 @@ ()
     'postgres-srcdir=s' => \$postgresSrcdir,
     'citus_abs_srcdir=s' => \$citusAbsSrcdir,
     'majorversion=s' => \$majorversion,
+    'worker-count=s' => \$workerCount,
```

Comment:
@rajeshkt78  used `i` here. I'm assuming that uses the variable as in integer?
```suggestion
    'worker-count=i' => \$workerCount,
```

Reply:
@JelteF  yes that is parse the argument as integer.